src/DaysAccordion.tsx

```diff
@@ -0,0 +1,242 @@
+          id="panel3bh-header"
+        >
+          <Typography>Wednesday</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell>Name</TableCell>
+                <TableCell align="right">Sets</TableCell>
+                <TableCell align="right">Reps</TableCell>
+                <TableCell align="right">Weight</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {exercises.wednesday.map((exercise: Exercise) => (
+                <TableRow key={exercise.id}>
+                  <TableCell>{exercise.name}</TableCell>
+                  <TableCell align="right">{exercise.sets}</TableCell>
+                  <TableCell align="right">{exercise.reps}</TableCell>
+                  <TableCell align="right">{exercise.weight}</TableCell>
+                </TableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </AccordionDetails>
+      </Accordion>
+      <Accordion defaultExpanded>
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="panel4bh-content"
+          id="panel4bh-header"
+        >
+          <Typography>Thursday</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell>Name</TableCell>
+                <TableCell align="right">Sets</TableCell>
+                <TableCell align="right">Reps</TableCell>
+                <TableCell align="right">Weight</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {exercises.thursday.map((exercise: Exercise) => (
+                <TableRow key={exercise.id}>
+                  <TableCell>{exercise.name}</TableCell>
+                  <TableCell align="right">{exercise.sets}</TableCell>
+                  <TableCell align="right">{exercise.reps}</TableCell>
+                  <TableCell align="right">{exercise.weight}</TableCell>
+                </TableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </AccordionDetails>
+      </Accordion>
+      <Accordion defaultExpanded>
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="panel4bh-content"
+          id="panel4bh-header"
+        >
+          <Typography>Friday</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell>Name</TableCell>
+                <TableCell align="right">Sets</TableCell>
+                <TableCell align="right">Reps</TableCell>
+                <TableCell align="right">Weight</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {exercises.friday.map((exercise: Exercise) => (
```

Comment:
This function that is given to `map` is duplicated 7 times. Good opportunity to extract it into its own function.